retail-managment-system/rms-ui/src/app/pages/supplier/supplier.component.scss

```diff
@@ -0,0 +1,53 @@
+.content {
+  height: 600px;
+}
+.btn {
+  background-color: #002d40;
+  color: white;
```

Comment:
if it exist in bootstrap why write style sheet

Reply:
for specific style

Reply:
if you are used bootstrap, it will not work?